### PR TITLE
fi-721 replace instances of class with local_class in terminology validation

### PR DESCRIPTION
**Pull request by czh-orz**
This fixes an error when running the encounter sequence. Now it should pass without any errors.

**Submitter:**
- [X] This pull request describes why these changes were made
- [X] Internal ticket for this PR: https://oncprojectracking.healthit.gov/support/browse/FI-721
- [X] Internal ticket links to this PR
- [X] Internal ticket is properly labeled (Community/Program)
- [X] Internal ticket has a justification for its Community/Program label
- [X] Code diff has been reviewed for extraneous/missing code
- [X] Tests are included and test edge cases
- [X] Tests/code quality metrics have been run locally and pass

**Reviewer 1:**

Name:
- [ ] Code is maintainable and reusable, reuses existing code and infrastructure
      where appropriate, and accomplishes the task's purpose
- [ ] The tests appropriately test the new code, including edge cases
- [ ] You have tried to break the code

**Reviewer 2:**

Name:
- [ ] Code is maintainable and reusable, reuses existing code and infrastructure
      where appropriate, and accomplishes the task's purpose
- [ ] The tests appropriately test the new code, including edge cases
- [ ] You have tried to break the code

### CODE REVIEW

File: generator/uscore/uscore_generator.rb [hunk 1]
```diff
@@ -703,6 +703,10 @@ def create_resource_profile_test(sequence)
         bindings = sequence[:bindings]
           .select { |binding_def| ['required', 'extensible'].include? binding_def[:strength] }
 
+        bindings.each do |binding|
+          binding[:path].gsub!('.class', '.local_class')
+          binding[:path] = 'local_class' if binding[:path] == 'class'
```
Comment (Jammjammjamm) [inline generator/uscore/uscore_generator.rb]: Only one of these should be necessary.

Comment (czh-orz) [reply generator/uscore/uscore_generator.rb]: fixed. Replaced with a better gsub expression

**Comment by arscan** [comment]
@czh-orz could you rebase and fix the conflict?  I'm not familiar enough w/what this SHOULD be.

**Comment by czh-orz** [comment]
rebased
